packages/react-core/src/demos/TextInputGroupDemo.md

```diff
@@ -24,7 +24,7 @@ Additionally, attributes can be selected by typing the full (case sensitive) nam
 
 Attributes can be deselected (returning you to attribute selection mode) by hitting `escape`, or by hitting `backspace` when the only text in the text input is the attribute.
 
-```js file="./examples/TextInputGroup/AttributeValueFiltering.js"
+```js file="./examples/TextInputGroup/AttributeValueFiltering.tsx"
```

Comment:
`ts file` I think

Reply:
hm... it may work both ways? aren't .ts files just for files with type declarations? I think .tsx could be correct? Now I'm second guessing everything lol